### PR TITLE
Added an API to retrieve information of all connected devices of a user

### LINKED ISSUE #732: Create an endpoint to return list of all connected devices for a single user
As discussed in the meeting, the list of connected devices are supposed to be displayed on the web client. An endpoint to store the information of connected devices has already been made (https://github.com/fossasia/susi_server/pull/728). A new endpoint is required which can now retrieve the stored information which can then be used later for displaying on the web client.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by Akshat-Jain**
Fixes #732 

Changes: I've added a servlet whose endpoint, that is /aaa/getAllDevices.json, returns the information of all of the connected devices (Smart Speakers) of a particular user in a JSON format. This retrieved information can then be used to display the list of connected devices on the web client (as discussed in the meeting).

### CODE REVIEW

**Comment by Akshat-Jain** [comment]
@anshumanv @akshatnitd @arundhati24 @batbrain7 @imjog Please review :-)

**Comment by Akshat-Jain** [comment]
@saurabhjn76 @dynamitechetan @DravitLochan Please review :-)

File: src/ai/susi/server/api/aaa/GetAllDevices.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        DAO.severe(e);
+      }
+
+      obj.put("accepted", true);
+      obj.put("message", "Information of all devices received!");
+      
+      return new ServiceResponse(obj);
+
+    }
+}
```
Comment (anshumanv) [inline src/ai/susi/server/api/aaa/GetAllDevices.java]: Add EOL

Comment (Akshat-Jain) [reply src/ai/susi/server/api/aaa/GetAllDevices.java]: Done, please check :-)

File: src/ai/susi/server/api/aaa/GetAllDevices.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+public class GetAllDevices extends AbstractAPIHandler implements APIHandler{
+
+    @Override
+    public String getAPIPath() {
+        return "/aaa/getAllDevices.json";
+    }
+
+    @Override
+    public UserRole getMinimalUserRole() {
+        return UserRole.ANONYMOUS;
```
Comment (dynamitechetan) [inline src/ai/susi/server/api/aaa/GetAllDevices.java]: What we need from this api is that a user can see what all devices are connected from his account. So this endpoint access role must not be anonymous. Set it to user level authorization.

**Pull request merged**